<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGCSettingsWidget</class>
 <widget class="QDialog" name="QGCSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <attribute name="toolTip">
       <string>General Settings</string>
      </attribute>
      <layout class="QGridLayout" name="generalPaneGridLayout">
       <property name="verticalSpacing">
        <number>8</number>
       </property>
       <item row="2" column="0">
        <widget class="QCheckBox" name="lowPowerCheckBox">
         <property name="toolTip">
          <string>Lowers all update rates to save battery power</string>
         </property>
         <property name="text">
          <string>Enable low power mode</string>
         </property>
         <property name="icon">
          <iconset resource="../../qgroundcontrol.qrc">
           <normaloff>:/files/images/status/battery-caution.svg</normaloff>:/files/images/status/battery-caution.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_message">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="styleType" stdset="0">
          <string notr="true">warning</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <layout class="QFormLayout" name="formLayout">
         <property name="horizontalSpacing">
          <number>8</number>
         </property>
         <property name="verticalSpacing">
          <number>8</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_language">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Language</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_language">
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_style">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Visual Style</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox_style">
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_style_custom">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Custom CSS:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="hLayout_style_custom">
           <item>
            <widget class="QLineEdit" name="lineEdit_customStylePath">
             <property name="placeholderText">
              <string>Select a file...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_selectCustomStyle">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../../qgroundcontrol.qrc">
               <normaloff>:/files/images/actions/folder-horizontal-open.png</normaloff>:/files/images/actions/folder-horizontal-open.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="checkBox_loadDefaultStyle">
           <property name="text">
            <string>Load default base style first.</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" alignment="Qt::AlignTop">
        <widget class="QCheckBox" name="audioMuteCheckBox">
         <property name="text">
          <string>Mute all audio output</string>
         </property>
         <property name="icon">
          <iconset resource="../../qgroundcontrol.qrc">
           <normaloff>:/files/images/status/audio-volume-muted.svg</normaloff>:/files/images/status/audio-volume-muted.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" alignment="Qt::AlignTop">
        <widget class="QCheckBox" name="reconnectCheckBox">
         <property name="text">
          <string>Automatically reconnect last Serial link on startup (not recommended)</string>
         </property>
         <property name="icon">
          <iconset resource="../../qgroundcontrol.qrc">
           <normaloff>:/files/images/devices/network-wireless.svg</normaloff>:/files/images/devices/network-wireless.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="showPerspectiveBtns">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select to show the Pilot/Mission/Config/Data buttons in the main toolbar.  Hide them to create more room for status display, then use the Perspectives menu or use the Alt-key shortcuts (Alt-P, Alt-M, Alt-C, and Alt-D, respectively).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Show perspective switch buttons on toolbar</string>
         </property>
         <property name="icon">
          <iconset resource="../../qgroundcontrol.qrc">
           <normaloff>:/files/images/actions/ui-toolbar.png</normaloff>:/files/images/actions/ui-toolbar.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qgroundcontrol.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QGCSettingsWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QGCSettingsWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
